src/index.js

```diff
@@ -142,7 +140,7 @@ function withAcornBigInt(acorn, Parser) {
   }
 }
 
-module.exports = function(Parser) {
+export default function numericSeparator(Parser) {
   const acorn = Parser.acorn || require("acorn")
```

Comment:
We probably should get rid of `require("acorn")` bit in ESM, and make only a CommonJS wrapper import it implicitly. Or, bump a major version, and get rid of implicit import altogether.

Reply:
Makes sense.